keyboards/grid/grid_650/readme.md

```diff
@@ -0,0 +1,68 @@
+# grid 650
+
+![grid 650](https://i.imgur.com/RGKxNq5h.jpg)
+
+[grid 650]( (https://www.gridishere.com/work/grid650)) is a 65% custom mechanical keyboard. 
+
+- Keyboard Maintainer: gridishere@gmail.com
+- Hardware Support: grid 650 QMK PCB V1 and V1, wired and BLE.
+
+
+
+### Compile
+
+Currently, only wired firmware can be compiled automatically.
+
+BLE firmware utilizes a [modified BLE driver](https://drive.google.com/file/d/1IBksQkk_YI8ERxd1O2vNbvkEjU2pdw5E/view?usp=share_link) ,which means you would need to download the modified driver files and replace the original one before compiling.
```

Comment:
Let me guess, requires an external library that isn't compatible with GPL? 

If so, please remove references to the BLE stuff.

Reply:
It does not require an external library, just need to add a line in the bluefruit_le.cpp under folder drivers/bluetooth. Do you think I still need to remove this?